packages/p2p/src/components/introducing-floating-rates-modal/introducing-floating-rates-modal.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { observer } from 'mobx-react-lite';
+import { Button, Checkbox, Modal, Text } from '@deriv/components';
+import { localize, Localize } from 'Components/i18next';
+import { getUrlBase, isMobile } from '@deriv/shared';
+import { useStores } from 'Stores';
+
+const IntroducingFloatingRatesModal = () => {
+    const { floating_rate_store, general_store } = useStores();
+    const should_not_show_modal_again = React.useRef(false);
+    const [is_open, setIsOpen] = React.useState(() => {
+        const p2p_settings = general_store.getLocalStorageSettings();
+        const show_introduce_fr_notf = p2p_settings[general_store.client.loginid]?.show_introduce_fr_notf;
+        return show_introduce_fr_notf ?? true;
+    });
+
+    const closeModal = () => {
+        setIsOpen(false);
```

Comment:
could we instead just directly put setIsOpen(false) as the function, rather than placing it within closeModal